src/sdk/model/vpg/vpg.ts

```diff
@@ -471,4 +473,16 @@ export class Vpg extends Entity {
     const observable: Observable<string> = Iland.getAuthProvider().getTokenObservable();
     return observable.map(token => `${href}&oauth_token=${token}`);
   }
+
+    /**
+     * Get performance counters for vpg
```

Comment:
last thing.. we try to standardize on ending the description in jsdoc with a period.